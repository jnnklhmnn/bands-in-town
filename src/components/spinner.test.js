import React from 'react';
import Spinner from './Spinner.js';
import renderer from 'react-test-renderer';

test('Spinner should match Snapshot', () => {
    const component = renderer.create(
        <Spinner />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});