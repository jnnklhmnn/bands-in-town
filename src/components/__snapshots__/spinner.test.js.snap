// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should match Snapshot 1`] = `
<div
  className="spinner"
>
  <div
    className="rect1"
  />
  <div
    className="rect2"
  />
  <div
    className="rect3"
  />
  <div
    className="rect4"
  />
  <div
    className="rect5"
  />
</div>
`;
